numba/dppl/ocl/atomics/atomic_ops.cl

```diff
@@ -0,0 +1,216 @@
+#ifdef cl_khr_int64_base_atomics
```

Comment:
Some comments should be added. If we are replicating what llvm-spirv does then a link to upstream will be helpful.

Reply:
Comment added